### PR TITLE
[BUGFIX] Fix dashboard widgets for v12

### LINKED ISSUE #475: [BUG] Widgets broken in v12
## Bug Report

**Current Behavior**
Widgets of type `BarChartWidget` break the dashboard with the error message

> **ArgumentCountError**
>
>TYPO3\CMS\Dashboard\Widgets\BarChartWidget::__construct(): Argument #3 ($backendViewFactory) not passed

Furthermore the `PaymentPaidShippingOpen` is broken when at least one item should be shown (works if it is empty). It only shows "An error occurred while retrieving information for this widget. Please try again, and if the problem persists, contact your administrator. " in this case.

**Expected behavior/output**
All widgets are working

### PR DESCRIPTION
**Pull request by rintisch**
* BarChartWidgets need a BackendViewFactory
which is added now.
* Fix `PaymentPaidShippingOpen` widget

Fixes #475

### CODE REVIEW

**Comment by rintisch** [comment]
@extcode The violation that is marked in the coding guideline is fixed in #477. Best to merge that first, afterwards I will rebase this commit.

File: Configuration/TypoScript/setup.typoscript [hunk 1]
```diff
@@ -387,4 +387,4 @@ plugin.tx_form.settings.yamlConfigurations {
 
 module.tx_form.settings.yamlConfigurations {
     1543219101 = EXT:cart/Configuration/Yaml/FormSetup.yaml
-}
+}
```
Comment (extcode) [inline Configuration/TypoScript/setup.typoscript:390]: It seems you removed the newline at end of file.

File: Configuration/page.tsconfig [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Template path for dashboard widgets
+templates.typo3/cms-dashboard.1712899110 = extcode/cart:Resources/Private/
```
Comment (extcode) [inline Configuration/page.tsconfig:2]: Can you add a newline here?

File: Configuration/TypoScript/setup.typoscript (hunk 1 above)
Comment (rintisch) [reply Configuration/TypoScript/setup.typoscript]: Done.

File: Configuration/page.tsconfig (hunk 2 above)
Comment (rintisch) [reply Configuration/page.tsconfig]: Done.

**Review by extcode — APPROVED** [review]

**Pull request merged**